chore: update user guide for release 25.09 (#933)
eclipse-tractusx/puris: docs/user/User_Guide.md

A user may use the page to send notifications to partners or read received notifications.

- `OPEN` for grouped messages that have at least one message open
- `RESOLVED` for grouped messages that have all been resolved

The notifications are grouped into collapsible sections based on their source disruption. Each collapsible displays the cause as well as effect of the disruption and can be expanded to display the list of notifications in table form.
The table displays `Outgoing` if the message was sent to a partner and `Incoming` if it was received from a partner.
If the notification has been resolved, the row will be grayed out and the text column will additionally display the Resolution Message.

### Notification View

![Overview of the notification view](img/notification_view.png)

One may get further information by clicking on a notification in the list.

### Notification Detail Modal

![Detailed notification after clicking the entry](img/notification_detail.png)

When triggering the button "NEW NOTIFICATION", a modal dialog is opened allowing a user to compose the demand and capacity notification. This creates a grouped section with the notification visible in the table.

> ### Send notification modal

Reviewer: ```suggestion
### Send Notification Modal
```

Please change to Title Case

- (optional) Affected Material Numbers (only those applicable to the partner)
- (optional) Affected Sites Recipient
- Text

If an outgoing notification has the status `Open` the user can interact with it by either editing it or resolving it.
When triggering the edit button the same modal is opened as for the new notification, the fields pre-populated with existing data.

### Edit Notification Modal

![Edit notification modal](img/notification_edit.png)

When the resolve button is triggered a modal dialog is opened allowing a user to write a resolution message and when saving will automatically change the status of the notification to `Resolved`.

### Resolve Notification Modal

![Resolve notification modal](img/notification_resolution.png)

If there is an outgoing open notification a user can resolve it by triggering the "Resolve" inline button (checkmark icon). This opens the resolution modal with a mandatory resolution message. Once submitted the status of the ticket is changed to resolved and cannot be edited. The text column is updated to contain the resolution message as well.

### Forward Notification Modal

![Forward notification modal](img/notification_forward.png)

> A user can also create a notification related to one the existing open groups by triggering the "FORWARD" button. This opens the creation modal.

Reviewer: ```suggestion
A user can also create a notification related to one of the existing open groups by triggering the "FORWARD" button. This opens the creation modal.
```

Sorry didn't notice the missing preposition before :/

- (optional) Affected Sites Recipient

> - (optional) Text

Reviewer: Text is not optional in these cases. (Can't suggest something for unchanged lines)

> If an Outgoing notification has the Open status the user can interact with it by either editing it or resolving it.
> When triggering the edit button the same modal is opened as for the New Notification, the fields pre-populated with existing data.

Reviewer: ```suggestion
If an outgoing notification has the status `Open` the user can interact with it by either editing it or resolving it.
When triggering the edit button the same modal is opened as for the new notification, the fields pre-populated with existing data.
```

I would opt to lowercase these words. As IT we tend to write Entities or Concepts uppercase (as it's like a name for us). For the users this might be confusing if not explicitly shown or sated (e.g. why should status open be written Uppercase? Yeah, makes sens if you refer to the value) :)

> ### Edit notification modal

Reviewer: ```suggestion
### Edit Notification Modal
```
Please change to Title Case

![Edit notification modal](img/notification_edit.png)

When the resolve button is triggered a modal dialog is opened allowing a user to write a resolution message and when saving will automatically change the status of the notification to `Resolved`.

> ### Resolve notification modal

Reviewer: ```suggestion
### Resolve Notification Modal
```
Please change to Title Case

![Resolve notification modal](img/notification_resolution.png)

> A user can also create a notification related to one the existing Open groups by triggering the "FORWARD" button. This opens the creation modal.

Reviewer: ```suggestion
A user can also create a notification related to one the existing open groups by triggering the "Resolve" inline button (checkmark icon). This opens the creation modal.
```

Reviewer: I also think the whole text should be under h3 "Forward Notification Modal" + here should be a description for resolving incl. mandatory information (one field).

The following fields are pre-selected and read only:

- Leading cause (same as the related notifications)
- Effect (same as the related notifications)
- Status (this field is automatically pre-selected with the `Open` option)

The Partner field only provides options for partners that aren't linked to any of the related or any of the existing outgoing notifications. This creates another entry in the existing table.

> ### Forward notification modal

Reviewer: ```suggestion
### Forward Notification Modal
```
Apply Title Case

![Forward notification modal](img/notification_forward.png)

> 

Reviewer: Please add description from above incl. mandatory information and mandatorynes, defaulting / restrictions

## Import

> An Admin can upload data in this page. Once the file is uploaded, all existing data for that type will be replaced by the newly uploaded data. They can click and download one of pre-existing template files listed on the page:

Reviewer: ```suggestion
An Admin can upload data in this page. Once the file is uploaded, all existing data for that type will be replaced by the newly uploaded data. This action only takes place if the file does NOT contain any invalid information. All infvalid rows are listed to the user with the respective validation error messages. Users can click and download one of pre-existing template files listed on the page:
```
